Upstream: http://calibre.kovidgoyal.net/ticket/1502
Source: philantrop@exherbo.org
Reason: Without this patch valuable metadata is lost.

diff -Naur src.orig/calibre/library/cli.py src/calibre/library/cli.py
--- src.orig/calibre/library/cli.py    2008-12-13 01:40:23.000000000 +0100
+++ src/calibre/library/cli.py    2008-12-18 07:01:17.000000000 +0100
@@ -76,13 +77,22 @@
   <entry>
       <title>${record['title']}</title>
       <id>urn:calibre:${record['id']}</id>
-      <author><name>${record['authors']}</name></author>
+      <author><name>${record['author_sort']}</name></author>
       <updated>${record['timestamp'].strftime('%Y-%m-%dT%H:%M:%SZ')}</updated>
       <link type="application/epub+zip" href="${quote(record['fmt_epub'].replace(sep, '/')).replace('http%3A', 'http:')}" />
       <link py:if="record['cover']" rel="x-stanza-cover-image" type="image/png" href="${quote(record['cover'].replace(sep, '/')).replace('http%3A', 'http:')}" />
       <link py:if="record['cover']" rel="x-stanza-cover-image-thumbnail" type="image/png" href="${quote(record['cover'].replace(sep, '/')).replace('http%3A', 'http:')}" />
       <content type="xhtml">
-          <div xmlns="http://www.w3.org/1999/xhtml">${record['comments']}</div>
+          <div xmlns="http://www.w3.org/1999/xhtml">
+          Authors: ${record['authors']}<br />
+          Publisher: ${record['publisher']}<br />
+          Rating: ${record['rating']}<br />
+          Tags: ${record['tags']}<br />
+          Series: ${record['series']}, book ${record['series_index']}<br />
+          ISBN: ${record['isbn']}<br />
+          <br />
+          ${record['comments']}
+          </div>
       </content>
   </entry>
   </py:for>
